ompi/mpiext/affinity/c/mpiext_affinity_str.c

```diff
@@ -107,10 +300,10 @@ static int get_rsrc_ompi_bound(char str[OMPI_AFFINITY_STRING_MAX])
 
     hwloc_bitmap_t cpuset = hwloc_bitmap_alloc();
     hwloc_bitmap_list_sscanf(cpuset, opal_process_info.cpuset);
-    if(OPAL_ERR_NOT_BOUND == opal_hwloc_base_cset2str(str,
-                                   OMPI_AFFINITY_STRING_MAX,
-                                   opal_hwloc_topology,
-                                   cpuset))
+    if(-1 == cset2str(str,
```

Comment:
This function can return more errors than just -1.  Are you only looking for the specific -1 error case here?  If so, a named enum would be better.  Or if you're just looking for non-success, then maybe check for != OMPI_SUCCESS?

(same with other places that call `cset2str()`)